batch/parsers/testSuiteParserUtils.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+
+export function normalizeTime(rawTime) {
+	let returnValue = 0;
+	if (rawTime) {
+		returnValue = Number.parseFloat(rawTime);
+	}
+	return returnValue.toFixed(2);
+}
+
+function parsedIntOrZero(stringToParse) {
+	return Number.parseInt(stringToParse) || 0;
+}
+export function testSuiteSummary(raw) {
+	return {
+		'tests': parsedIntOrZero(raw.tests),
+		'failures': parsedIntOrZero(raw.failures),
+		'skipped': parsedIntOrZero(raw.skipped),
+		'errors': parsedIntOrZero(raw.errors)
```

Comment:
I wonder if this does not obscure the code rather than make it clearer (well, that is of course, unless we assume everyone is perfectly familiar w/ `unary +`)